modules/ROOT/pages/customize.adoc

```diff
@@ -0,0 +1,107 @@
+= Customize the Look and Feel of Your API Community
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Use the https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank] to customize the branding, look and feel, and site structure of your community pages to match your desired brand identity.
+
+Select *Community Builder* in the API Community Manager (ACM) administrator interface. By default, the
+Community Builder displays the *Home* page and the Pages drop-down list displays *Home*. You can select
+the Pages drop-down list to display a list of community pages and specify a different page for customization.
+
+The *Community Builder* interface contains the customization icons described here.
+
+== Components
+
+Select the *Components* lightning bolt icon in the top left navigation to display the component palette.
+The component palette includes Salesforce Community Cloud and ACM-specific components.
+Drag and drop the desired components from the component palette to the community page. Each
+component has its own configuration options that you can then access by selecting the component on the page.
+
+For detailed component information, refer to xref:acm-lightning-components.adoc[ACM Lightning Components] and
+the Salesforce Community Cloud https://help.salesforce.com/articleView?id=rss_component_reference_table.htm&type=5[Community Components, role=external, window=blank] documentation.
+
+Components that are typically used in *Home* pages include:
+
+* https://help.salesforce.com/articleView?id=rss_headline.htm&type=5[Headline, role=external, window=blank]
+* https://help.salesforce.com/articleView?id=rss_setup_nav_menu.htm&type=5[Navigation Menu, role=external, window=blank]: You can update *Max Items to Display (2-15)*,
+*Menu Item Alignment*, and use the *Navigation Menu* to display, add, remove, and customize options. *Home*
+is always present and is always the first item. The following properties can be customized:
```

Comment:
Suggest moving "*Home* is always present and is always the first item" to earlier text, because it sounds like it is the first thing that should be said about community customization rather than between customizing display, align, and nav. options.
Also, what is the connection between these three options and "the following properties" that can be customized? What are these properties of?

Did I hear you guys on a call yesterday, with Meeta saying that we don't want to include screen captures because of fluidity and translation? Too bad...they would be good here.

Reply:
One other thing to consider in terms of this topic: does it provide much useful content outside of providing all those links? Should the linked info actually be included in the overviews to components, themes, pages, and settings?

Reply:
The Navigation Menu component always has a link **Home** to the Home page, and it's always the first item on the menu. The other links can be added, removed, reordered, and customized with the properties Name, Type, Page, and Publicly Available. Rewrote this to clarify.

The top level of the builder UI offers the menus Components, Theme, Pages, and Settings. The Components menu offers all the components described, such as Headline, Navigation Menu, CMS Collection, Rich text editor, and HTML editor. I want the structure of info in this page to fit the UI, and I've added more text explaining how all of these controls appear in the builder UI.

Screen captures are used in some places, and this may be a worthwhile place. Added that idea to the list.